Hand_Off_Lab39_for_HRF.MD

```diff
@@ -13,26 +13,27 @@
 </br>
 
 ## Where Are the Potentially Useful Codes?
-> Please check archive.py
+> Please check archive.py.
 </br>
 </br>
 </br>
 
 ## Issues with the Current Version of the Codebase:
 1. ### Multiple tweets entries in the database may have the same incident ID
-> The current codebase may allow the option to send the tweet with the form so that many may provide the missing information regarding the incident. This means that we may have multiple input for the same incident ID, which would in turn populate the database as separate, stand-alone entries. 
+> The current codebase may allow the option to send the tweet with the form so that many may provide the missing information regarding the incident. This means that we may have multiple input for the same incident ID, which would in turn presents to the admin. dashboard as stand-alone entries. 
```

Comment:
which would in turn be presented to the admin via the dashboard as stand-alone entries.